evap/staff/views.py

```diff
@@ -717,7 +717,6 @@ def course_create(request, semester_id):
             raise SuspiciousOperation("Invalid POST operation")
 
         course = course_form.save()
-        course.set_last_modified(request.user)
         course.save()
```

Comment:
form.save already saves the model, so the model.save shouldn't be necessary anymore (there's quite a few instances of that pattern)